packages/engine-api-client/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@lodestar/engine-api-client",
+  "version": "1.2.1",
+  "description": "Package for interacting with the ethereum execution layer",
+  "type": "module",
+  "author": "ChainSafe Systems",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/lodestar.git"
+  },
+  "exports": "./lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
check the other package.jsons, need a `typesVersions` here

Reply:
This should fix that type issue where you're needing `/src`

Reply:
> This should fix that type issue where you're needing `/src` in the imports

the issue ended up being something else...The`typesVersion` has also been added as suggested